docs/backend-requests/handling/nodejs.mdx

```diff
@@ -1,7 +1,198 @@
+
+  const port = process.env.PORT || 3000;
+
+  const app: Application = express();
+
+  declare global {
+    namespace Express {
+      interface Request extends LooseAuthProp {}
+    }
+  }
+
+  // Use the lax middleware that returns an empty auth object when unauthenticated
+  app.get(
+    '/protected-route',
+    ClerkExpressWithAuth({
+      // ...options
+    }),
+    (req: WithAuthProp<Request>, res: Response) => {
+      res.json(req.auth);
+    }
+  );
+
+  app.use((err, req, res, next) => {
+    console.error(err.stack);
+    res.status(401).send('Unauthenticated!');
+  });
+
+  app.listen(port, () => {
+    console.log(`Example app listening at http://localhost:${port}`);
+  });
+  ```
+</CodeBlockTabs>
+
+## `ClerkExpressRequireAuth()`
+
+<CodeBlockTabs options={['JavaScript', 'TypeScript']}>
+  ```js
+  import "dotenv/config"; // To read CLERK_API_KEY
+  import { ClerkExpressRequireAuth } from '@clerk/clerk-sdk-node';
+  import express from 'express';
+
+  const port = process.env.PORT || 3000;
+  const app = express();
+
+  // Use the strict middleware that raises an error when unauthenticated
+  app.get(
+    '/protected-endpoint',
+    ClerkExpressRequireAuth({
+      // ...options
+    }),
+    (req: RequireAuthProp<Request>, res) => {
+      res.json(req.auth);
+    }
+  );
+
+  app.use((err, req, res, next) => {
+    console.error(err.stack);
+    res.status(401).send('Unauthenticated!');
+  });
+
+  app.listen(port, () => {
+    console.log(`Example app listening at http://localhost:${port}`);
+  });
+
+  
+  ```
+
+  ```typescript
+  import "dotenv/config"; // To read CLERK_API_KEY
+import {
+  ClerkExpressRequireAuth,
+  RequireAuthProp,
+  StrictAuthProp,
+} from '@clerk/clerk-sdk-node';
+import express, { Application, Request, Response } from 'express';
+
+const port = process.env.PORT || 3000;
+const app: Application = express();
+
+declare global {
+  namespace Express {
+    interface Request extends StrictAuthProp {}
+  }
+}
+
+// Use the strict middleware that raises an error when unauthenticated
+app.get(
+  '/protected-route',
+  ClerkExpressRequireAuth({
+    // ...options
+  }),
+  (req: RequireAuthProp<Request>, res) => {
+    res.json(req.auth);
+  }
+);
+
+app.use((err, req, res, next) => {
+  console.error(err.stack);
+  res.status(401).send('Unauthenticated!');
+});
+
+app.listen(port, () => {
+  console.log(`Example app listening at http://localhost:${port}`);
+});
+
+  ```
+</CodeBlockTabs>
+
+### Express Error Handlers
+Express comes with a default error handler for errors encountered in the middleware chain.
+
+Developers can also implement their own custom error handlers as detailed in the [Express error handling guide](https://expressjs.com/en/guide/error-handling.html). An example error handler can be found above.
+
+If you are using the strict middleware variant, the err pass to your error handler will contain enough context for you to respond as you deem fit.
+
+## Middleware options
+
+<Tables
```

Comment:
There is no complex component usage so a `<Table />` component is unnecessary.

For better readability, use a markdown table.

E.g.

| Name | Type | Description |
| --- | --- | --- |
| `authorizedParties` | `string[]` | Validate that the `azp` claim in the Clerk Session JWT equals any of your known Origins that are permitted to generate those tokens. This is an extra security check that we highly recommend that you do. For more information, refer to the [Manual JWT Verification](/docs/backend-requests/handling/manual-jwt) documentation.

```suggestion
<Tables
```